README.md

```diff
@@ -0,0 +1,123 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [AWESOME BOOKS] <a name="about-project"></a>
+
+AWESOME BOOKS project in Microverse.
+
+**[AWESOME_BOOKS]** is a project experience that Microverse offers to help nuture your skills in designing on dynamic webpages.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+ 3. JavaScript
+
+
+### Key Features <a name="key-features"></a>
+
+  1. prints out a book store webpage
+  2. It is made to cater for mobile devices and desktops
+  3. Menu bar that open up to give various options
+  4. contact section
+  5. Responsive design
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [Awesome Books](https://github.com/MargaretTechworld/Awesome-Books.git)
+### Install
+
+Install this project with:
+
+cd MY-PORTFOLIO
+npm install
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+👤 **MargaretTechworld**
+
+- GitHub: [MargaretTechworld](https://github.com/MargaretTechworld)
+- Twitter: [KojoMargaret](https://twitter.com/KojoMargaret)
+- LinkedIn: [MargaretTechworld](https://www.linkedin.com/in/margaret-kojo-musa-759b8026a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+1: Details modal 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+##👍 Contributors
+- GitHub: [Okari Nyandika](https://github.com/MargaretTechworld/Awesome-Books/commits?author=2004-okari)
+
+Feel free to check the [issues page](https://github.com/MargaretTechworld/Awesome-Books/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can star the repository to make it look better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me all the necessary knowledge i need to make this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
Great job team 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.